src/client/runner/automation/playback/press/index.js

```diff
@@ -149,4 +173,14 @@ export default class PressAutomation {
     }
 }
 
+// Setup cross-iframe interaction
+//TODO: move to top
+messageSandbox.on(messageSandbox.SERVICE_MSG_RECEIVED_EVENT, e => {
+    if (e.message.cmd === PRESS_REQUEST_CMD) {
+        hammerhead.on(hammerhead.EVENTS.beforeUnload, () => messageSandbox.sendServiceMsg({ cmd: PRESS_RESPONSE_CMD }, e.source));
```

Comment:
we have two case to send response: `beforeUnload` event and successful press execution. I'd like to store both in one function. Do we really need `onPressRequest` function? Maybe we can place the logic here? Or move binding on `beforeUnload` to `onPressRequest` (I'd prefer the first)